src/noos_inv/python.py

```diff
@@ -103,6 +104,8 @@ def package(ctx: Context, install: str | None = None) -> None:
         ctx.run("poetry build", pty=True)
     if install_type == InstallType.pipenv:
         ctx.run("pipenv run python -m build -n", pty=True)
+    if install_type == InstallType.uv:
+        ctx.run("uvx --from build pyproject-build --installer uv", pty=True)
```

Comment:
🐼 

Invoking `uv` commands through a pseudo terminal isn't necessary as `uv` is compiled at source and not a Python script.

https://www.pyinvoke.org/faq.html#why-is-my-command-behaving-differently-under-invoke-versus-being-run-by-hand

`ctx.run("uvx --from build pyproject-build --installer uv")` should suffice.